packages/config/tests/context/fixtures/legacy_functions_directory/netlify.toml

```diff
@@ -0,0 +1,7 @@
+[build]
```

Comment:
It seems like this fixture directory is not referenced anywhere.

Reply:
Ah, yes! I added this fixture for another branch still being developed and it incorrectly made its way to this PR. Removed in https://github.com/netlify/build/pull/2447/commits/68888190af0a485392bae5d08e56b881fb177878.